docs/docs/workflows/recovery.md

```diff
@@ -87,6 +87,15 @@ If this fails due to an unhealthy control plane, you will see log messages simil
 
 This means that you have to recover the node manually.
 
+</tabItem>
+<tabItem value="aws" label="AWS">
+
+:::caution
+
+Recovery isn't yet implemented for AWS. If you require this feature, [let us know](https://github.com/edgelesssys/constellation/issues/new?assignees=&labels=&template=feature_request.md)!
```

Comment:
Is there actually anything blocking us from supporting recovery?
As far as I know, this should just require us to set up a loadbalancer the CLI talks tom the rest should be CSP independent.

Reply:
@3u13r is working on the loadbalancer part. I can take another stab at recovery once it's merged.

Reply:
Tested this with with a single control-plane cluster, and a 3 control-plane nodes cluster.
Both self recovery (at least half of the control-planes are still reachable) and manually recovery using the CLI work on AWS.

Reply:
We still need to manually pass the IP to `constellation recover`, right? So I see two ways forward:
1. Keep saying that `constellation recover` is not supported
2. Document the way of getting the right IP from AWS and recovering with that. 

I have no real opinion on which way we choose.